client/src/client-only-routes/ShowSettings.test.js

```diff
@@ -14,7 +14,8 @@ describe('<ShowSettings />', () => {
     expect(navigate).toHaveBeenCalledWith(
       `${apiLocation}/signin?returnTo=settings`
     );
-    expect(true).toBeTruthy();
+    const result = shallow.getRenderOutput();
+    expect(result.type.displayName).toBe('Loader');
```

Comment:
What happens when we have a user?

Reply:
Would you like me to have loggedinprops and test it?

Reply:
Yup, that is one of the critical renders I believe, like the email renders or not correctly